clients/src/main/resources/common/message/FetchResponse.json

```diff
@@ -47,33 +47,35 @@
       "about": "The top level response error code." },
     { "name": "SessionId", "type": "int32", "versions": "7+", "default": "0", "ignorable": false,
       "about": "The fetch session ID, or 0 if this is not part of a fetch session." },
-    { "name": "Topics", "type": "[]FetchableTopicResponse", "versions": "0+",
+    { "name": "Responses", "type": "[]FetchableTopicResponse", "versions": "0+",
```

Comment:
s/Responses/TopicResponses?

Reply:
I was trying to keep the field names aligned with what was defined in the manual schemas. So, not sure if we should change any other field names in this PR